Sources/PromotedCore/ErrorHandling/Errors.swift

```diff
@@ -163,8 +172,43 @@ extension MetricsLoggerError: NSErrorProperties {
       return 402
     case .unexpectedEvent(_):
       return 403
-    case .missingUserIDsInUserMessage:
+    case .missingLogUserIDInUserMessage:
```

Comment:
Reading these numbers again, they look almost like http response codes.  It's probably fine.  JFYI that it caught me off guard a little.  It's probably fine.  If customer devs can see these codes in a different way, it might make sense to use a different range.

Reply:
Also a good point. Let me bump them up to 4 digits.